nominees/pharmacovigilance-electronic-reporting-tool.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Pharmacovigilance Electronic Reporting Tool",
+  "aliases": [
+    "PvERS II"
+  ],
+  "description": " It is web and mobile based solution which provides reporting and surveillance functionalities of  individual case study reports (ICSR) and consumer reporting in pharmacovigilance.",
+  "website": "https://web.pharmacyboardkenya.org/pharmacovigilance/",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://www.gnu.org/licenses/gpl-3.0.en.html"
```

Comment:
Invalid license URL. This should be a link to where the license is indicated for this project's repository. Request project to include a license file to the main [repo](https://github.com/IntelliSOFT-Consulting/pvers). Thanks.

Reply:
Resolved in commit fef1314